matchbox_socket/src/ggrs_socket.rs

```diff
@@ -6,13 +6,14 @@ impl WebRtcSocket {
     /// Returns a Vec of connected peers as [`ggrs::PlayerType`]
     #[must_use]
     pub fn players(&self) -> Vec<PlayerType<String>> {
+        let client_id = self.get_id().unwrap().expect("Client has no ID.");
```

Comment:
This method either needs to be documented that it's invalid to call before having an id, or it should return a result